Gruntfile.js

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  'use strict';
+
+  var pkg = grunt.file.readJSON('package.json');
+
+  grunt.initConfig({
+    pkg: pkg,
+    pkgReadOnly: pkg
+  });
+  // load all grunt tasks matching the `grunt-*` pattern
+  require('load-grunt-tasks')(grunt);
+  // load local Grunt tasks
+  grunt.loadTasks('tasks');
+  // this one is def not ready yet
+  grunt.registerTask('build',
```

Comment:
Since you are loading the tasks from the `tasks` subdirectory, you might as well create two new files for the `build` and `lint` tasks to keep things consistent.